.github/workflows/website-production.yml

```diff
@@ -0,0 +1,95 @@
+name: WebsiteProduction
+
+defaults:
+  run:
+    working-directory: website
+
+on:
+  workflow_dispatch:
+    inputs:
+      deploy:
+        description: 'type "deploy" to confirm deploy to production (main branch only)'
+        required: false
+  push:
+    branches: [main]
+    paths:
+      - .github/workflows/website.yml
+      - website/**
+      - packages/snack-sdk/**
+      - .eslint*
+      - .prettier*
+      - yarn.lock
+  pull_request:
+    branches: [main]
+    paths:
+      - .github/workflows/website.yml
+      - website/**
+      - packages/snack-sdk/**
+      - .eslint*
+      - .prettier*
+      - yarn.lock
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Decrypt secrets
+        uses: sliteteam/github-action-git-crypt-unlock@a09ea5079c1b0e1887d4c8d7a4b20f00b5c2d06b
+        env:
+          GIT_CRYPT_KEY: ${{ secrets.GIT_CRYPT_KEY }}
+
+      - name: Install Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@94337306dda8180d967a56932ceb4ddcf01edae7
+        with:
+          project_id: exponentjs
+          service_account_key: ${{ secrets.GCLOUD_KEY }}
+          export_default_credentials: true
+
+      - name: Install Google Cloud components
+        run: gcloud components install kubectl kustomize skaffold
+
+      - name: Configure Google Cloud SDK
+        run: |
+          gcloud config set project exponentjs
+          gcloud config set compute/zone us-central1-f
+          gcloud config set container/cluster exp-central
+          gcloud container clusters get-credentials exp-central
+          gcloud auth configure-docker gcr.io
```

Comment:
I copied this from the CircleCI config in `universe`. I'm not entirely sure whether this is needed.